client/scripts/app.ts

```diff
@@ -863,48 +868,17 @@ Promise.all([
   initAppState(true, customDomain).then(async () => {
     // setup notifications and websocket if not already set up
     if (!app.socket) {
-      let jwt;
-      // refresh notifications once
       if (app.loginState === LoginState.LoggedIn) {
+        // refresh notifications once
         app.user.notifications.refresh().then(() => m.redraw());
-        jwt = app.user.jwt;
```

Comment:
Why do we remove the jwt?

Reply:
Still using it from `app.user.jwt`, but the variable jwt is redundant. If you're not logged in then you can't receive notifications anyways so the WebSocket connection should fail (if `app.user.jwt` is undefined).